domain-model/src/main/java/uk/gov/hmcts/cmc/domain/amount/TotalAmountCalculator.java

```diff
@@ -53,6 +54,21 @@ public static BigDecimal calculateInterest(
         );
     }
 
```

Comment:
please arrange public , private and overloaded methods accordingly

Reply:
overloaded methods needs to be next to each other regardless of their visibility, otherwise codacy will raise this.